### PR TITLE
"unofficial/3rd party support" annotations about Service Mesh Interface compatibility

### LINKED ISSUE #41: Istio's Service Mesh Interface Compatibility (except for metrics for now)
Now the website is saying simply "YES".

And the reason behind the answer "YES" is [smi-adapter-istio](https://github.com/servicemeshinterface/smi-adapter-istio) implementation.

But I guess it's slightly misleading because of following issues.
https://github.com/servicemeshinterface/smi-adapter-istio/issues/94
https://github.com/servicemeshinterface/smi-adapter-istio/issues/93
https://github.com/servicemeshinterface/smi-adapter-istio/issues/74

If there is no problem, I will try to add some annotations to these rows.

### PR DESCRIPTION
**Pull request by tkms0106**
close #41

### CODE REVIEW

**Comment by tkms0106** [comment]
Any problems? If there are, I will fix.

File: _includes/comparison.html [hunk 1]
```diff
@@ -244,7 +244,7 @@ <h2 class="section-heading--primary"> Service Mesh Comparison </h2>
     <tbody>
       <tr>
         <th scope="row">Traffic Access Control</th>
-        <td><a href="https://github.com/servicemeshinterface/smi-adapter-istio" target="_blank">yes</a></td>
+        <td><a href="https://github.com/servicemeshinterface/smi-adapter-istio" target="_blank">yes (inofficial/3rd party support)</a></td>
```
Comment (erikdw) [inline _includes/comparison.html:247]: inofficial -> unofficial (everywhere)

**Comment by HPrinz** [comment]
Thank you @tkms0106 for your contribution and sorry this took so long.
Do you agree with the proposed changes by @erikdw?

**Comment by HPrinz** [comment]
Thanks a lot!

**Pull request merged**